src/NHPersistenceCompatibilityTests/Common/NugetHelper.cs

```diff
@@ -2,25 +2,31 @@
 using System.Collections.Generic;
 using System.Linq;
 using System;
+using System.Configuration;
 
 namespace Common
 {
     public class NugetHelper
     {
-        string packageSource;
-        string fallbackPackageSource;
+        IEnumerable<string> packageSources;
 
-        public NugetHelper(string packageSource = "https://packages.nuget.org/api/v2", string fallbackPackageSource = "https://www.myget.org/F/particular/")
+        public NugetHelper()
         {
-            // Todo: Load this package source from the machine config
-            // Todo: Support multiple package sources (e.g. Build server with Myget and Nuget)
-            this.packageSource = packageSource;
-            this.fallbackPackageSource = fallbackPackageSource;
+            LoadNugetSourcesFromConfig();
+        }
+
+        private void LoadNugetSourcesFromConfig()
+        {
+            var machineSettings = Settings.LoadDefaultSettings(new PhysicalFileSystem("C:\\"), null, null);
```

Comment:
@WilliamBZA I see, those types are from the Nuget assembly.

Would any of the paths apply mentioned here: https://docs.nuget.org/consume/nuget-config-file

So that would be `%APPDATA%\NuGet\NuGet.Config` or is it a different kind of path?

If that is the path then please adjust it. If this is a different kind of path then leave it.